src/components/AudioCell.vue

```diff
@@ -50,6 +52,10 @@
       @keyup.escape.exact="blurCell"
       @mouseup.right="onRightClickTextField"
     >
+      <template v-slot:error>
+        文章が長いため正常に動作しない可能性があります。
+        文章を分割することを推奨します。
```

Comment:
迷いましたが、エラーなのでエラーメッセージっぽく指示を出すようにするのはどうでしょう。
```suggestion
        句読点の位置で文章を分割してください。
```